src/common/filter/Expressions.h

```diff
@@ -37,6 +37,8 @@ struct Getters {
 
 class ExpressionContext final {
 public:
+    explicit ExpressionContext(std::string spaceCollate) : spaceCollate_(spaceCollate) {}
```

Comment:
A suggestion, set a default value of param spaceCollate.

Reply:
When default values are set, no parameters are passed during construction, which may result in different charset.